gene2phenotype_project/gene2phenotype_app/models.py

```diff
@@ -203,28 +231,15 @@ class Meta:
 class LGDMolecularMechanismEvidence(models.Model):
     id = models.AutoField(primary_key=True)
     molecular_mechanism = models.ForeignKey("LGDMolecularMechanism", on_delete=models.PROTECT)
-    description = models.TextField(null=True)
+    evidence = models.ForeignKey("CVMolecularMechanism", on_delete=models.PROTECT, default=None)
```

Comment:
Suggesting we add related_name here

Reply:
The related_name is necessary when we have more than one FK pointing to the same table.
In this case we only have evidence pointing to cv_molecular_mechanism.